### DISCUSSION TITLE
Remove url slug to further shorten url

### DISCUSSION THREAD

**Discussion by kennedy** [ClearURLs/Rules#91] [Ideas]
Sometimes urls have human readable slugs but its not necessary because the id of the page is also provided. 

Examples:

- https://www.amazon.com/LG-UltragearTM-Monitor-nVIDIA®-Compatible/dp/B0BRBW8KRK -> https://www.amazon.com/dp/B0BRBW8KRK
- https://www.bestbuy.com/site/lg-ultragear-27-oled-qhd-240hz-0-03ms-freesync-and-nvidia-g-sync-compatible-gaming-monitor-with-hdr10-black/6530357.p?skuId=6530357 -> https://www.bestbuy.com/site/6530357.p

How would I go about adding this option? would it be under `rawRules`?

**Comment by kennedy** [comment]
I made a feature request issue here: https://github.com/ClearURLs/Rules/issues/90

But I figured a discussion would be a good place to start first

**Comment by stokito** [comment]
The Clean URL is intended to reduce tracking but this feature is not improving this. This can create problems with caching. It also creates an additional point of failure. What is a use case for the feature?
